Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -485,3 +485,121 @@ public extension TransposedConv2D {
             padding: padding)
     }
 }
+
+/// A 2-D depthwise convolution layer for spatial depthwise convolution over images.
+///
+/// This layer creates seperable convolution filters that are convolved with the layer input to produce a
+/// tensor of outputs.
+@frozen
+public struct DepthwiseConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The 4-D convolution kernel for the depthwise convolution.
+    public var depthwise_kernel: Tensor<Scalar>
+    /// The bias vector.
+    public var bias: Tensor<Scalar>
+    /// An activation function.
+    public typealias Activation = @differentiable (Tensor<Scalar>) -> Tensor<Scalar>
+    /// The element-wise activation function.
+    @noDerivative public let activation: Activation
+    /// The strides of the sliding window for spatial dimensions.
+    @noDerivative public let strides: (Int, Int)
+    /// The padding algorithm for convolution.
+    @noDerivative public let padding: Padding
+
+    /// Creates a `DepthwiseConv2D` layer with the specified filter, bias, activation function, strides, and
+    /// padding.
+    ///
+    /// - Parameters:
+    ///   - depthwise_kernel: The 4-D convolution kernel.
+    ///   - bias: The bias vector.
+    ///   - activation: The element-wise activation function.
+    ///   - strides: The strides of the sliding window for spatial dimensions.
+    ///   - padding: The padding algorithm for convolution.
+    public init(
+        depthwise_kernel: Tensor<Scalar>,
+        bias: Tensor<Scalar>,
+        activation: @escaping Activation,
+        strides: (Int, Int),
+        padding: Padding
+    ) {
+        self.depthwise_kernel = depthwise_kernel
+        self.bias = bias
+        self.activation = activation
+        self.strides = strides
+        self.padding = padding
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The output.
+    @differentiable
+    public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+        return activation(depthwiseConv2D(input, depthwise_kernel: depthwise_kernel,
+                                 strides: (1, strides.0, strides.1, 1),
```

Comment:
Align line-wrapped arguments with the first argument on the previous line.
```suggestion
                                          strides: (1, strides.0, strides.1, 1),
```